HHS/Head-Start-TTADP
# [Prod] Update README; Smartsheet integration prototype, updated side navigation

## Description of change
- Added non-production Smartsheet integration prototype.
- Updated README.
- Fixed a bug where removing the only goal from a report and clicking save draft would not save the goal.
- Update the side navigation for the site, grouping like elements into expandable sections
- TTAHUB-2726: Two column view is shown for Goal and Objectives.
- TTAHUB-3057: All not started objectives for goal 66089.

## How to test
- To see the Smartsheet integration, navigate to the Admin section under the SS tab. Verify that the data is there.
- Confirm the side navigation works appropriately.
- TTAHUB-2726: Make sure two column view is shown for Goal and Objectives.
- TTAHUB-3057: All not started objectives for goal 66089.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2307
* https://jira.acf.gov/browse/TTAHUB-2982
* https://jira.acf.gov/browse/TTAHUB-2396
* https://jira.acf.gov/browse/TTAHUB-2726
* https://jira.acf.gov/browse/TTAHUB-3057

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
